source/install/troubleshooting.rst

```diff
@@ -30,6 +30,56 @@ Important notes
 - Any manipulation of the database must be done using the built in command line tools.
 - Start simple with the step-by-step install guides for your operating system.
 
+Deployment troubleshooting
+--------------------------
+
+Docker deployments
+~~~~~~~~~~~~~~~~~~
+
+If you're deploying the Mattermost server using Docker on an M1 Mac and encountering permission issues in the Docker container, `re-reate the required directories and set their permissions <https://docs.mattermost.com/guides/deployment.html#deploy-mattermost-for-production-use>`__, then skip the following command:
```

Comment:
```suggestion
If you're deploying the Mattermost server using Docker on an M1 Mac and encountering permission issues in the Docker container, `re-create the required directories and set their permissions <https://docs.mattermost.com/guides/deployment.html#deploy-mattermost-for-production-use>`__, then skip the following command:
```

Reply:
At what point does the command that must be skipped occur and is it a command that has a Y/N/Skip option?

Reply:
On #5517, I proposed the troubleshooting tip differently. This should say:

> If deploying on an M1 Mac and encountering permission issues in the Docker container, redo the third step and skip this command:
> 
> `sudo chown -R 2000:2000 ./volumes/app/mattermost`

On M1, this permission change causes the deploy to not work. It's not something you can skip with a y/n/skip, but rather you omit it.